app/components/global_threads/thread_list/thread_list_header/index.tsx

```diff
@@ -62,13 +62,13 @@ function ThreadListHeader({haveUnreads, intl, markAllAsRead, style, testID, view
             </View>
             <View style={style.markAllReadIconContainer}>
                 <TouchableOpacity
-                    disabled={!haveUnreads}
+                    disabled={false/*!haveUnreads*/}
```

Comment:
Why? Is then the prop not really needed?

Reply:
https://mattermost.atlassian.net/browse/MM-37495

As we are looking for a temporary solution, thought commenting would be a quick fix.